src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
```

Comment:
Давай створимо інтерфейс `Drawable` з методом `draw()` і імплементуємо його фігурам
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```